Cargo.toml

```diff
@@ -51,7 +51,7 @@ serde_json = "1.0.82"
 # TODO: Update after https://github.com/fwcd/tree-sitter-kotlin/pull/71 lands
 tree-sitter-kotlin = { git = "https://github.com/ketkarameya/tree-sitter-kotlin.git", rev = "a87ddd003368e068563f1cc478a1b2a3f9d73b60" }
 tree-sitter-java = "0.20.0"
-tree-sitter-swift = "0.3.4"
```

Comment:
Let's make an issue against the upstream repo and add it as a todo like we have done for kotlin/go